frontend/src/charts/ChoroplethMap.tsx

```diff
@@ -513,12 +512,13 @@ export default function ChoroplethMap(props: ChoroplethMapProps) {
   )
 
   return (
-    <Grid
-      container
-      justifyContent={'center'}
+    <div
+      className='justify-center'
       ref={props.overrideShapeWithCircle ? undefined : ref}
-      sx={{ mt: props.isUnknownsMap ? 5 : 0 }}
-      display={width === INVISIBLE_PRELOAD_WIDTH ? 'none' : undefined}
+      style={{
+        marginTop: props.isUnknownsMap ? 5 : 0,
+        display: width === INVISIBLE_PRELOAD_WIDTH ? 'none' : undefined,
+      }}
```

Comment:
can these be moved to tailwind as well?

like `className={props.isUnknownsMap ? 'mt-5' : 'mt-0'}` and similar?

Reply:
gotcha, done.